app/livechat/server/api/v1/contact.js

```diff
@@ -0,0 +1,37 @@
+import { Match, check } from 'meteor/check';
+
+import { API } from '../../../../api/server';
+import { Contacts } from '../../lib/Contacts';
+
+
+API.v1.addRoute('contact', { authRequired: true }, {
+	post() {
```

Comment:
`POST` method is used to create new objects, which means the `ID` field is not expected here. If you want to allow updating objects, you need to use `PUT`.